### PR TITLE
Add support for right handed coordinate systems

### PR DESCRIPTION
**Pull request by ckeiner**
HtmlMesh doesn't work with a right handed coordinate system. This pull request aims to provide support for such coordinate systems.

### CODE REVIEW

**Comment by sebavan** [comment]
Lets ask @ericwood73 to review ???

**Comment by ckeiner** [comment]
I added support for billboardMode = 1 and also added a few more screens to see rotations for various angles. This should give a good indication of pretty much all cases that are interesting.

If @ericwood73 has some time, it'd be great to get some feedback :)

**Review by ericwood73 — COMMENTED** [review]
Changes look reasonable.  I'd like to pull down and build a new package so the changes can be confirmed in the playground.  Will do that today and if there are no issues, will indicate approval @sebavan

File: HtmlMesh/example.js
Comment (ericwood73) [inline HtmlMesh/example.js:1]: @ckeiner Would it be a good idea to have an example-rh.js that applies the same transformations as the original example using a RH system so the two examples look identical and we can confirm that changing to RH didn't change anything other than the way you specify your transforms?  Then we could just comment in or out example.js/example-rh.js in index.html to test and confirm.

Comment (sebavan) [reply HtmlMesh/example.js:1]: @ericwood73 let me know once all good and I will merge it :-)

File: HtmlMesh/example-rhs.js [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        </form>
+    `;
+    div.style.backgroundColor = "white";
+    div.style.width = "480px";
+    div.style.height = "360px";
+
+    htmlMeshDiv.setContent(div, 4, 3);
+    htmlMeshDiv.position.x = 3;
+    htmlMeshDiv.position.y = 2;
+    htmlMeshDiv.rotation = new Vector3(Math.PI / 4, Math.PI / 4, Math.PI / 4);
```
[the remark this answers is not in the document]
Comment (ckeiner) [reply HtmlMesh/example-rhs.js]: @ericwood73 Yep, worked . Thanks :)

image

**Comment by ericwood73** [comment]
Thank you for the contribution @ckeiner.  @sebavan These changes look good to me.

**Review by sebavan — APPROVED** [review]

**Pull request merged**